src/resources/js/tickets.js

```diff
@@ -8,6 +8,11 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 	var $tribe_tickets                   = $( document.getElementById( 'tribetickets' ) );
 	var $tickets_container               = $( document.getElementById( 'event_tickets' ) );
 	var $post_id                         = $( document.getElementById( 'post_ID' ) );
+
+	var $metabox                         = $tribe_tickets.find( '.inside' );
```

Comment:
This variable seems...extraneous? While it narrows down the later searches some, they could all be run from `$tribe_tickets` - no?

You could even do `$tribe_tickets.find( '.inside > .tribe-tickets-editor-block' );`

Up for discussion! 😄

Reply:
Yeah we could mostly run it from `$tribe_tickets` Since it's done only once in a very narrow scope. It allows for better control later on if we need.